package/src/renderer/__tests__/e2e/Snapshot.spec.tsx

```diff
@@ -24,4 +24,7 @@ describe("Snapshot", () => {
   itRunsE2eOnly("should respect overflow: hidden", async () => {
     await testSnapshot("Snapshot4");
   });
+  itRunsE2eOnly("should respect ScrollView offset and padding", async () => {
```

Comment:
the reference results are missing right?

Reply:
Another modification to make test simpler could be to remove scrolling from test so it will at least test overflow issue.